File: ProyectoCsharp/Modelo/Proveedor.cs
public class Proveedor{
    private long idProveedor;
	private string RUC;
	private string razon_social;
	private string email;
	private string telefono;



    //Constructores
    public Proveedor(){
		RUC = razon_social = email = telefono = null;
	}
	
	public Proveedor(long id_proveedor, string RUC, string razon_social, string email, string telefono){
        IdProveedor = id_proveedor;
        RUC1 = RUC;
        Razon_social = razon_social;
        Email = email;
        Telefono = telefono;
	}
	
	//Destructores
	~Proveedor(){
		RUC = razon_social = email  = telefono = null;
	}

    //Metodos Get y Set

    public long IdProveedor { get => idProveedor; set => idProveedor = value; }
    public string RUC1 { get => RUC; set => RUC = value; }
    public string Razon_social { get => razon_social; set => razon_social = value; }
    public string Email { get => email; set => email = value; }
    public string Telefono { get => telefono; set => telefono = value; }
}

File: TrabajoJavaLp2/src/modelo/Proveedor.java
package modelo;
public class Proveedor{
        private long id_proveedor;
	private String ruc;
	private String razon_social;
	private String email;
        private String direccion;
	private String telefono;

        public Proveedor(){
            id_proveedor = 0;
            ruc = null;
            razon_social = null;
            email = null;
            direccion = null;
            telefono = null;
	}
        
        public Proveedor(long id_proveedor, String ruc, String razon_social, String email, String direccion, String telefono) {
            this.id_proveedor = id_proveedor;
            this.ruc = ruc;
            this.razon_social = razon_social;
            this.email = email;
            this.direccion = direccion;
            this.telefono = telefono;
        }

        public long getId_proveedor() {
            return id_proveedor;
        }

        public void setId_proveedor(long id_proveedor) {
            this.id_proveedor = id_proveedor;
        }
        
	public void setRUC(String ruc){
		this.ruc = ruc;
	}

	public String getRUC(){
		return ruc;
	}

	public void setRazon_social(String razon_social){
		this.razon_social = razon_social;
	}

	public String getRazon_socual(){
		return razon_social;
	}

	public void setEmail(String email){
		this.email = email;
	}

	public String getEmail(){
		return email;
	}
        
        public String getDireccion() {
            return direccion;
        }

        public void setDireccion(String direccion) {
            this.direccion = direccion;
        }

	public void setTelefono(String telefono){
		this.telefono = telefono;
	}

	public String getTelefono(){
		return telefono;
	}
}